pkg/database/config.go

```diff
@@ -15,8 +16,13 @@ type Config struct {
 	Username string `mapstructure:"username"`
 	Password string `mapstructure:"password"`
 	Database string `mapstructure:"database"`
-	Pool     int    `mapstructure:"pool"`
 	Timeout  string `mapstructure:"timeout"`
+	// Connection settings
+	// TODO Pool field name must be modified in the next major change.
+	Pool                  int           `mapstructure:"pool"`
```

Comment:
[q]: Do you mean that it should be renamed to `MaxIdleConnections` (`max_idle_connections`)? If so, maybe is a good idea to create a card for this?

We can even make it backwards compatible:

```go
// Deprecated: Use MaxIdleConnections instead.
Pool int `mapstructure:"pool"`

MaxIdleConnections int `mapstructure:"max_idle_connections"`
```

Reply:
Yes, exactly. I can create a ticket, no problem, but wouldn't it overload our backlog with bunch of 1 point tickets?

Reply:
tbh, the change is super easy (with marking the `Pool` as deprecated), we could even do this without a Jira card in a separate PR (or even a separate commit in this PR 🙊). And we may release the new version of the `go-sdk` together with these changes.

My main point here, having a `TODO` as an inline comment does not help us here because most likely no one will notice that